Core/GDCore/IDE/Events/EventsRefactorer.h

```diff
@@ -208,6 +209,19 @@ class GD_CORE_API EventsRefactorer {
                                      gd::String newString,
                                      bool matchCase);
 
+  /**
+   * Replace all occurrences of a gd::String in string events (comments and
+   * groups)
+   *
+   * \return true if something was modified.
+   */
+  static bool ReplaceStringInStringEvent(gd::ObjectsContainer& project,

```

Comment:
```suggestion
  static bool ReplaceStringInEventStrings(gd::ObjectsContainer& project,
```

or even `ReplaceStringInEventSearchableStrings` to be very precise, but I'll leave the choice up to you!